Dockerfile

```diff
@@ -34,4 +45,6 @@ ENV NODE_ENV production
 ENV COMMIT_HASH $COMMIT_HASH
 ENV BUILD_DATE $BUILD_DATE
 
-CMD ["tini", "--", "yarn", "start"]
+RUN echo "DATABASE_URL=$(node dist/scripts/print-database-url.js)" > .env
```

Comment:
we shouldn't bake the DATABASE_URL into the image, as it potentially exposes a secret, won't work for the images published publicly on Docker Hub, and makes it harder to change

I think it should work without this step?

Reply:
Now it can't expose any secret because location of file with SQLite DB isn't a secret.

But I changed it to be more configurable.